Source/WebCore/Modules/modern-media-controls/controls/macos-fullscreen-media-controls.js

```diff
@@ -77,6 +77,9 @@ class MacOSFullscreenMediaControls extends MediaControls
 
         this.bottomControlsBar.element.addEventListener("mousedown", this);
         this.bottomControlsBar.element.addEventListener("click", this);
+        this.element.addEventListener("mousemove", function() {
+            this.faded = false;
+        }.bind(this));
```

Comment:
This does not follow the coding style used throughout the media controls codebase. You should pass `this` as the event listener and route the `mousemove` event through in the `handleEvent()` method as we do with `mousedown` and `click`.

Reply:
You'll be able to query `this._lastDragPoint` to check you're not in the controls dragging code initiated under `_handleMousedown()`.

Reply:
I think I fixed this in the new upload